xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -173,7 +174,7 @@ namespace PVR
       if (item->GetPVRRecordingInfoTag())
         return CServiceBroker::GetPVRManager().GUIActions()->ShowRecordingInfo(item);
 
-      return CServiceBroker::GetPVRManager().GUIActions()->ShowEPGInfo(item);
+      return CServiceBroker::GetPVRManager().Get<PVR::GUI::EPG>().ShowEPGInfo(item);
```

Comment:
A general question is this form of `Get<...>()` going to be use to get any component from the PVR Manager, GUI component or otherwise?

Reply:
> going to be used to get any component from the PVR Manager, GUI component

 yes
 
 > If so how would one know what the overall set of classes are that this can be called for?
 
  every class that is registered with `CPVRComponentRegistration`. Every class that implements `IPVRComponent` can be registered there.
  
  The "PVR::GUI::..." notation is just a `using` for the real type (e.g. PVR::CPVRGUIActionsParantalControl). See the respective PVRGUIAction... header files.